modulemd/tests/test-modulemd-packager-v3.c

```diff
@@ -313,252 +306,8 @@ packager_test_map_to_stream_v2 (void)
   g_clear_pointer (&yaml_str, g_free);
   g_clear_pointer (&expected_path, g_free);
   g_clear_pointer (&expected_str, g_free);
-
-  /* restore default mdversion to avoid unexpected results from other tests */
-  modulemd_set_default_stream_mdversion (default_mdv);
-}
-
-static void
```

Comment:
Except for the bits about setting/restoring the default_stream_mdversion, this test is still valid.